assets/javascripts/discourse/templates/list/topic-meta.raw.hbs

```diff
@@ -1,16 +1,28 @@
+{{#unless tilesStyle}}
 <div class="topic-meta">
   <div class='topic-category inline sub'>
     {{raw-plugin-outlet name="topic-list-social-meta-category"}}
     {{category-link topic.category}}
+    <span class="middot crazy inline sub"></span>
   </div>
+{{/unless}}
   {{#if topic.tags}}
-    <span class="middot inline sub"></span>
+    {{#if tilesStyle}}
+    <div class="topic-tags">
```

Comment:
@merefield Formatting again here. Make sure all code is indented properly.

Reply:
I've reverted this file to the current HEAD.